docs-src/concepts/what-is-the-temporal-platform.md

```diff
@@ -9,12 +9,89 @@ tags:
 ---
 
 The Temporal Platform consists of a [Temporal Cluster](/concepts/what-is-a-temporal-cluster) and [Worker Processes](/concepts/what-is-a-worker-process).
-Together these components create a runtime for Workflow Executions.
+Together these components create a runtime for Workflow Executions; that is-they create a runtime for your application.
 
 ![The Temporal Platform](/diagrams/temporal-platform-simple.svg)
 
 The Temporal Cluster is open source and can be operated by you.
-The Temporal Cloud is a set of Clusters operated by us.
+[Temporal Cloud](/concepts/what-is-temporal-cloud) is a set of Clusters operated by us.
```

Comment:
For reasons not apparent to me, this sentence is rendering with a blank line above instead of below. Any ideas?

Reply:
Likely. Until we (by which I mean @flossypurse, of course) can figure out what's causing this behavior, we can cheat a little by putting both sentences in the same line.

Reply:
would even that work?